modules/apps/app-builder/app-builder-test/src/testFunctional/paths/AppBuilder.path

```diff
@@ -135,6 +135,42 @@
 	<td></td>
 </tr>
 
+<tr>
+	<td>DEPLOY_APP_STATUS</td>
+	<td>//span[contains(text(),'Deployed')]</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>UNDEPLOY_APP_BUTTON</td>
+	<td>//button[@class='btn btn-primary']</td>
+	<td></td>
+</tr>
+
+<tr>
+	<td>PRODUCT_MENU_DEPLOYED</td>
+<td>//span[@class='control-menu-level-1-heading text-truncate']</td>
+	<td></td>
```

Comment:
Hi @nkkwon , could you please ident the <td> tags that are not indented on the new locators you are adding?